db/migrate/20220809145955_add_author_id_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorIdToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_column :likes, :AuthorId, :integer
+    add_index :likes, :AuthorId
```

Comment:
- [ ] Also same here trying to add a column won't create the relationship between likes and the user for more information on how to solve that problem look into the above comment